gradoop-flink/src/test/java/org/gradoop/flink/model/GradoopFlinkTestBase.java

```diff
@@ -69,17 +72,17 @@
    */
   private GradoopFlinkConfig config;
 
-  private LogicalGraphLayoutFactory graphLayoutFactory;
+  private LogicalGraphLayoutFactory<GraphHead, Vertex, Edge> graphLayoutFactory;
```

Comment:
Since almost everything in this class got a java doc description i prefer to add one here as well.